tests/language_specific/english/test_english_language_generator.py

```diff
@@ -768,6 +768,24 @@ def test_bird_flies_up():
     assert generated_tokens(situation) == ("a", "bird", "flies", "up")
 
 
+def test_jump_up():
```

Comment:
I take it that this test and the one below it cover the two basic formats: jumping w/o an explicit goal location and jumping with a goal? If that's the case, these tests should be sufficient!